### PR TITLE
Add TOC for convenient navigation through the docs

### PR DESCRIPTION
**Pull request by bocharsky-bw**

### CODE REVIEW

**Comment by willdurand** [comment]
I would not add the full links but :+1:

**Comment by bocharsky-bw** [comment]
Good point, I'll make them relative

File: Resources/doc/index.md [hunk 1]
```diff
@@ -10,6 +10,22 @@ of this bundle. For version `1.x`, please read:
 Also, you might be interested in this [UPGRADE
 guide](https://github.com/willdurand/BazingaJsTranslationBundle/blob/master/UPGRADE.md).
 
+- [Installation](/blob/master/Resources/doc/index.md#installation)
```
Comment (stof) [inline Resources/doc/index.md]: that won't work due to the way Github handles relative links when rendering doc (it treats them as inside the repo, so `/blob/master` should not be there).

and as these are only links inside the same page, you could even simplify it using `#installation` only

Comment (bocharsky-bw) [reply Resources/doc/index.md]: You're right, thanks!

**Comment by monteiro** [comment]
👍

**Pull request merged**